server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/QuartzJobChangeReq.java

```diff
@@ -15,44 +15,33 @@
  */
 package com.oceanbase.odc.service.schedule.model;
 
-import java.util.Date;
+import org.quartz.JobDataMap;
 
-import com.oceanbase.odc.core.shared.constant.TaskStatus;
+import com.oceanbase.odc.service.quartz.model.MisfireStrategy;
 
 import lombok.Data;
-import lombok.NonNull;
 
 /**
  * @Author：tinker
- * @Date: 2023/5/24 15:35
+ * @Date: 2024/6/9 22:19
  * @Descripition:
  */
 
 @Data
-public class ScheduleTaskResp {
-
-    private Long id;
+public class QuartzJobChangeReq {
```

Comment:
ChangeQuartJobReq


and, will it be used for front-end request , if yes, avoid quartz concept?